debian/docs

```diff
@@ -0,0 +1,3 @@
+README.md
+CONTRIBUTING.md
+NOTICES
```

Comment:
NOTICES shouldn't be shipped I think. We have debian/copyright for that and the remainder is expressed in package dependencies.